src/components/QuickProjectSearch.tsx

```diff
@@ -56,21 +54,10 @@ export default function QuickProjectSearch({
     }
   }, [inputText])
 
-  const sepanaEnabled = featureFlagEnabled(FEATURE_FLAGS.SEPANA_SEARCH)
-
-  const { data: sepanaSearchResults, isLoading: isLoadingSepanaSearch } =
-    useSepanaProjectsSearch(searchText, {
-      pageSize: MAX_RESULTS,
-      enabled: sepanaEnabled,
-    })
-
-  const { data: graphSearchResults, isLoading: isLoadingGraphSearch } =
-    useProjectsSearch(searchText, { enabled: !sepanaEnabled })
-
-  const searchResults = sepanaEnabled ? sepanaSearchResults : graphSearchResults
-  const isLoadingSearch = sepanaEnabled
-    ? isLoadingSepanaSearch
-    : isLoadingGraphSearch
+  const { data: searchResults, isLoading } = useDBProjectsQuery({
```

Comment:
this removes previous feature flags and defaults the project search query to use the database